Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```diff
@@ -2021,6 +2021,112 @@ script:
     - contextPath: ServiceNow.Ticket.TagID
       description: Ticket tag ID.
       type: String
+  - arguments:
+    - default: false
+      description: Query by name. Match by contains and not by exact name.
```

Comment:
It can also be matched by exact name... this is a bit confusing.

Reply:
Agreed. I'll ask @amshamah419 to the rescue.